otx/algorithms/action/adapters/mmaction/models/backbones/movinet.py

```diff
@@ -0,0 +1,1048 @@
+"""
+Code inspired by:
+https://github.com/Atze00/MoViNet-pytorch/blob/main/movinets/models.py
+"""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from collections import OrderedDict
+from typing import Any, Callable, Optional, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from einops import rearrange
+from mmaction.models.builder import BACKBONES
```

Comment:
I think it makes sense to import this into mmcv's builder (mmaction.BACKBONES -> mmcv.BACKBONES).

Reply:
We use train_model function inside of mmaction and that function use builder from mmaction to build model.
Is it still okay to change registry?

Reply:
If you need more confirmation (mmaction.Registry vs mmcv.Registry), in that case, leave it like this for now.